google-home-community.groovy

```diff
@@ -178,6 +178,12 @@ def mainPreferences() {
                 type: "bool",
                 defaultValue: false
             )
+            input(
+                name: "immediateResponse",
+                title: "Respond to Google before device is at new state",
+                type: "bool",
+                defaultValue: false
+            )
```

Comment:
Is this setting necessary?  I feel like `immediateResponse=true` could just be the default and only option.

Reply:
This was in the original PR — I just kept it so we could compare the before/after behavior and have an escape valve in case I royally messed things up. :)

I agree the new behavior is better in almost every case, and I'm happy to either get rid of this setting or default it to on (give folks an escape valve in case something is messed up).

Reply:
I'd just get rid of it.  If something's messed up we can always fix it or roll back.

Reply:
OK, removed config setting and bumped version number to 0.30.0.